app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
provide type annotations here

Reply:
I see, you haven't resolved this issue yet, but asked for review.
**Please don't hesitate to write to mentor if u have some questions.**
btw Arsen means smth like that
`class Person:`
`    people: dict = {}`